sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -204,6 +205,23 @@ export class JsonRpcProvider extends Provider {
     return objects.filter((obj: SuiObjectInfo) => Coin.isSUI(obj));
   }
 
+  getCoinDenominationInfo(
+    coin_type: string,
+  ): CoinDenominationInfoResponse {
+    if (coin_type !== '0x2::sui::SUI') {
+      throw new Error(
+        'only SUI coin is supported in getCoinDenominationInfo for now.'
+      );
+    }
+    return {
+      coin_type: coin_type,
+      basic_unit: 'MIST',
+      canonical_unit: 'SUI',
+      denomination: BigInt(1000000000),
```

Comment:
can we also export this as a const MIST_IN_SUI

Reply:
as the other comment suggested removing this field, will remove it instead.